internal/mux/muxfunc.go

```diff
@@ -43,30 +33,12 @@ func MatchRange(lower, upper byte) MatchFunc {
 //              |    [128..191] -+--> forward to RTP/RTCP
 //              +----------------+
 
-// MatchSTUN is a MatchFunc that accepts packets with the first byte in [0..3]
-// as defied in RFC7983
-func MatchSTUN(b []byte) bool {
-	return MatchRange(0, 3)(b)
-}
-
-// MatchZRTP is a MatchFunc that accepts packets with the first byte in [16..19]
-// as defied in RFC7983
-func MatchZRTP(b []byte) bool {
-	return MatchRange(16, 19)(b)
-}
-
 // MatchDTLS is a MatchFunc that accepts packets with the first byte in [20..63]
 // as defied in RFC7983
 func MatchDTLS(b []byte) bool {
 	return MatchRange(20, 63)(b)
 }
 
-// MatchTURN is a MatchFunc that accepts packets with the first byte in [64..79]
-// as defied in RFC7983
-func MatchTURN(b []byte) bool {
```

Comment:
This and above functions seem unrelated to the PR, i assume they just weren't in use?

Reply:
Yea! I just deleted anything that was unused. Since it is in `/internal` we should be the only users.